tcms/testplans/templates/testplans/get.html

```diff
@@ -80,10 +80,12 @@ <h2 class="card-pf-title">
     <div class="row row-cards-pf">
         <div class="col-xs-12 col-sm-12 col-md-12">
             <div class="card-pf card-pf-accented">
-                <h2 class="card-pf-title">
-                    {% trans 'Confirmed test cases' %}
-                </h2>
```

Comment:
This is the same section which we've got in tc_executions.html. Why change ?

Reply:
I was opened the two pages(test case and test plan) and tried to make them both use same html structure, obviously I messed up. Its my bad, it should use h2 tag.